pkg/controller/node/nodecontroller.go

```diff
@@ -373,26 +381,29 @@ func (nc *NodeController) Run() {
 		defer nc.evictorLock.Unlock()
 		for k := range nc.zonePodEvictor {
 			nc.zonePodEvictor[k].Try(func(value TimedValue) (bool, time.Duration) {
-				obj, exists, err := nc.nodeStore.GetByKey(value.Value)
+				message, _ := value.UID.(evictionMessage)
```

Comment:
You can't do that. This is a rate limited queue, which is designed to limit number of Node evictions. After your change it's limiting Pod evictions, which is wrong. Please fix.